### PR TITLE
Install deepvariant

### PR DESCRIPTION
**Pull request by ramprasadn**
### This PR adds | fixes:

- Installs deepvariant

### How to test:

- Automatic and continuous test suite

### Expected outcome:
- [ ] Installation, unit and integration test suite pass

### Review:
- [ ] Code review
- [ ] Tests pass

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes. If applicable record manual test results in PR header
- [x] **MINOR** - when you add functionality in a backwards compatible manner. If applicable record manual test results in PR header
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: templates/program_test_cmds.yaml [hunk 1]
```diff
@@ -30,6 +30,11 @@ cnvnator:
   path: 'cnvnator'
 cyrius:
   execution: 'star_caller.py --help'
+deepvariant:
+  path: 'run_deepvariant'
+  path: 'make_examples'
+  path: 'call_variants'
+  path: 'postprocess_variants'
```
Comment (jemten) [inline templates/program_test_cmds.yaml:37]:
```suggestion
```

Comment (jemten) [reply templates/program_test_cmds.yaml:37]: At the moment it's only possible to test one program per container with MIP's installation test.

**Comment by ramprasadn** [comment]
Awesome! Will do!

**Comment by ramprasadn** [comment]
Thanks @henrikstranneheim!
Woot! First merge! :D

**Pull request merged**